### PR TITLE
Update Set-VMProcessor.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a tip for perfmon parameter. cannot add it as a switch as it is only available on intel Processors

### CODE REVIEW

**Comment by msbemba** [comment]
For issue#https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1809

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/hyper-v/Set-VMProcessor.md [hunk 1]
```diff
@@ -87,7 +87,8 @@ PS C:\> Set-VMProcessor TestVM -ExposeVirtualizationExtensions $true
 Configures virtual machine TestVM, enabling embedded Hyper-V.Uu
 
 
-
+> [!Tip]
+> To Enable Intel Performance Monitoring Hardware in a Hyper-V virtual machine you can use the -Perfmon parameter. For more information about requirements, visit [Enable Intel Performance Monitoring Hardware in a Hyper-V virtual machine](https://docs.microsoft.com/en-us/windows-server/virtualization/hyper-v/manage/performance-monitoring-hardware)
```
Comment (get-itips) [inline docset/windows/hyper-v/Set-VMProcessor.md:91]:
```suggestion
> To Enable Intel Performance Monitoring Hardware in a Hyper-V virtual machine you can use the -Perfmon parameter. For more information about requirements, visit [Enable Intel Performance Monitoring Hardware in a Hyper-V virtual machine](https://docs.microsoft.com/windows-server/virtualization/hyper-v/manage/performance-monitoring-hardware)
```

File: docset/windows/hyper-v/Set-VMProcessor.md [hunk 2]
```diff
@@ -87,7 +87,8 @@ PS C:\> Set-VMProcessor TestVM -ExposeVirtualizationExtensions $true
 Configures virtual machine TestVM, enabling embedded Hyper-V.Uu
 
 
-
+> [!Tip]
```
Comment (JohanFreelancer9) [inline docset/windows/hyper-v/Set-VMProcessor.md:90]:
```suggestion
> [!TIP]
```

File: docset/windows/hyper-v/Set-VMProcessor.md [hunk 3]
```diff
@@ -87,7 +87,8 @@ PS C:\> Set-VMProcessor TestVM -ExposeVirtualizationExtensions $true
...
+> To Enable Intel Performance Monitoring Hardware in a Hyper-V virtual machine you can use the -Perfmon parameter. For more information about requirements, visit [Enable Intel Performance Monitoring Hardware in a Hyper-V virtual machine](https://docs.microsoft.com/windows-server/virtualization/hyper-v/manage/performance-monitoring-hardware)
```
Comment (JohanFreelancer9) [inline docset/windows/hyper-v/Set-VMProcessor.md:91]:
```suggestion
> To enable Intel Performance Monitoring Hardware in a Hyper-V virtual machine, you can use the -Perfmon parameter. For more information about requirements, visit [Enable Intel Performance Monitoring Hardware in a Hyper-V virtual machine](https://docs.microsoft.com/windows-server/virtualization/hyper-v/manage/performance-monitoring-hardware).
```

**Comment by get-itips** [comment]
> I think we should add this parameter to the parameters section. What do you think? Why as a TIP?

Maybe adding it with a note.

**Comment by msbemba** [comment]
> > I think we should add this parameter to the parameters section. What do you think? Why as a TIP?
> 
> Maybe adding it with a note.

@dariomws  and @get-itips I was reluctant as this parameter is not available on all processors. Should I add a note, example and the description of the parameter stating that this is available only on Intel processors?

**Comment by dariomws** [comment]
@msbemba could you test this parameter?

I would bet that it is available in any Windows Server 2019 and Windows 10 Version 1809 or later, regardless of the processor, but it would only work if the requirements indicated in the article are met.
That being said, I think adding the parameter in the parameter section with a short description and the link would be the best option.

Even better if we can test it to be sure. I tested with my machine but I have Intel processor and it works.

**Comment by e0i** [comment]
Please tag me whenever the revisions are ready to go. @msbemba

**Comment by msbemba** [comment]
> @msbemba could you test this parameter?
> 
> I would bet that it is available in any Windows Server 2019 and Windows 10 Version 1809 or later, regardless of the processor, but it would only work if the requirements indicated in the article are met.
> That being said, I think adding the parameter in the parameter section with a short description and the link would be the best option.
> 
> Even better if we can test it to be sure. I tested with my machine but I have Intel processor and it works.

I have Intel processor as well. I will update the parameter section with a short description and the link.

**Pull request merged**